azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/implementation/ComputeManager.java

```diff
@@ -59,6 +97,13 @@ public AvailabilitySets availabilitySets() {
         return availabilitySets;
     }
 
+    /**
+     * Get Azure virtual machine collection.
+     * <p/>
+     * The collection supports performing CRUD operations on Azure virtual machines
+     *
+     * @return The virtual machine collection
```

Comment:
btw, something to thjkn about, I've read that a well accepted practice for javadoc'ing plain getters is to skip the top level description, and only provide the @return